hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/RocksDBStoreMBean.java

```diff
@@ -179,6 +234,7 @@ public void getMetrics(MetricsCollector metricsCollector, boolean b) {
     MetricsRecordBuilder rb = metricsCollector.addRecord(contextName);
     getHistogramData(rb);
     getTickerTypeData(rb);
+    getDBPropertyData(rb);
```

Comment:
When `ozone.metastore.rocksdb.statistics` is set to `OFF` (by default), these metrics won't be available.

Understand that `HistogramData` and `TickerTypeData` are retrieved from `rocksdb.statistics`, so when config if set to "OFF", metrics won't be visible. But seems `DBPropertyData` are independent, should we skip the check on the config?

Reply:
hey @ChenSammi ~ just a follow up on this!

Reply:
@symious ,  some newly added metrics, for example "estimate-pending-compaction-bytes". If the value is too high, it 's a warning message. To find out why there are so many pending compaction bytes, we need to look into other rocksdb metrics, which are exposed by HistogramData and TickerTypeData.  

So although the newly added properties in patch doesn't depend on rocksdb.statistics turn on, but in a real problem investigation, all these three types of metrics information are all needed. So rocksdb.statistics controls all metrics display is a fair solution.

Reply:
I'm not sure about this.

I reckon the default value of "ozone.metastore.rocksdb.statistics" was set to "false" because of some performance impact. But since the newly added metrics don't have such impacts, I think it's better to have these metrics exposed by default.